samples/cpp/select3dobj.cpp

```diff
@@ -19,48 +19,52 @@
 #include <ctype.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string>
 
 using namespace std;
 using namespace cv;
 
-const char* helphelp =
-"\nThis program's purpose is to collect data sets of an object and its segmentation mask.\n"
-"\n"
-"It shows how to use a calibrated camera together with a calibration pattern to\n"
-"compute the homography of the plane the calibration pattern is on. It also shows grabCut\n"
-"segmentation etc.\n"
-"\n"
-"select3dobj -w=<board_width> -h=<board_height> [-s=<square_size>]\n"
-"           -i=<camera_intrinsics_filename> -o=<output_prefix>\n"
-"\n"
-" -w=<board_width>          Number of chessboard corners wide\n"
-" -h=<board_height>         Number of chessboard corners width\n"
-" [-s=<square_size>]            Optional measure of chessboard squares in meters\n"
-" -i=<camera_intrinsics_filename> Camera matrix .yml file from calibration.cpp\n"
-" -o=<output_prefix>        Prefix the output segmentation images with this\n"
-" [video_filename/cameraId]  If present, read from that video file or that ID\n"
-"\n"
-"Using a camera's intrinsics (from calibrating a camera -- see calibration.cpp) and an\n"
-"image of the object sitting on a planar surface with a calibration pattern of\n"
-"(board_width x board_height) on the surface, we draw a 3D box around the object. From\n"
-"then on, we can move a camera and as long as it sees the chessboard calibration pattern,\n"
-"it will store a mask of where the object is. We get successive images using <output_prefix>\n"
-"of the segmentation mask containing the object. This makes creating training sets easy.\n"
-"It is best if the chessboard is odd x even in dimensions to avoid ambiguous poses.\n"
-"\n"
-"The actions one can use while the program is running are:\n"
-"\n"
-"  Select object as 3D box with the mouse.\n"
-"   First draw one line on the plane to outline the projection of that object on the plane\n"
-"    Then extend that line into a box to encompass the projection of that object onto the plane\n"
-"    The use the mouse again to extend the box upwards from the plane to encase the object.\n"
-"  Then use the following commands\n"
-"    ESC   - Reset the selection\n"
-"    SPACE - Skip the frame; move to the next frame (not in video mode)\n"
-"    ENTER - Confirm the selection. Grab next object in video mode.\n"
-"    q     - Exit the program\n"
-"\n\n";
-
+static string helphelp(char** argv)
+{
+    return  string("") +
```

Comment:
You can wrap the first part of message with `std::string()` and not introduce extra empty line.

Reply:
Thinks a lot. I will continue improving these problems.